include/crow/json.h

```diff
@@ -1675,7 +1675,7 @@ namespace crow
                 }
                 else
                 {
-#if defined(__APPLE__) || defined(__MACH__) || defined (__FreeBSD__)
+#if defined(__APPLE__) || defined(__MACH__) || defined (__FreeBSD__) || defined (__ANDROID__)
```

Comment:
This shouldn't be part of this PR i'm guessing

Reply:
I am sorry, this is a define I forgot to include in the last pull request for android.